lib/rex/ui/text/table.rb

```diff
@@ -274,14 +323,17 @@ def columns_to_s # :nodoc:
     last_idx = nil
     columns.each_with_index { |col,idx|
       if (last_col)
-        nameline << pad(' ', last_col, last_idx)
-
-        remainder = colprops[last_idx]['MaxWidth'] - last_col.length
-        if (remainder < 0)
-          remainder = 0
-        end
+        # This produces clean to_s output without truncation
+        # Preserves full string in cells for to_csv output
+        padding = pad(' ', last_col, last_idx)
+        nameline << padding
+        remainder = padding.length - cellpad
+      if (remainder < 0)
+        remainder = 0
+      end
```

Comment:
Agreed, not sure why this is formatted as such, believe i got some flak back in the day for getting too "clever" with ternary ops and other syntactic sugar which made my submissions look very inconsistent with upstream. Will update